README.rst

```diff
@@ -73,7 +73,10 @@ Versions within the Errata Tool.
         is_server_only: false
         rhel_release_name: RHEL-8
         sig_key_name: redhatrelease2
+        container_sig_key_name: redhatrelease2
         allow_buildroot_push: false
+        use_quay_for_containers: False
+        use_quay_for_containers_stage: False
```

Comment:
Same here, please remove this since it's already `False` by default (and then we don't have to lower-case this to satisfy `ansible-lint`)

Reply:
Done!